src/compiler/parser.ts

```diff
@@ -9788,7 +9808,7 @@ namespace IncrementalParser {
         if (sourceFile.statements.length === 0) {
             // If we don't have any statements in the current source file, then there's no real
             // way to incrementally parse.  So just do a full parse instead.
-            return Parser.parseSourceFile(sourceFile.fileName, newText, sourceFile.languageVersion, /*syntaxCursor*/ undefined, /*setParentNodes*/ true, sourceFile.scriptKind, sourceFile.setExternalModuleIndicator);
+            return Parser.parseSourceFile(sourceFile.fileName, newText, sourceFile.languageVersion, /*syntaxCursor*/ undefined, /*setParentNodes*/ true, sourceFile.scriptKind, sourceFile.setExternalModuleIndicator, sourceFile.jsdocSkipped);

```

Comment:
Here in the incremental parser, I'm pretty sure I need to verify that the two jsdocSkipped flags are identical, otherwise things will break? I haven't done that here but just realized it.

Reply:
No, this flag won't change, nevermind.